src/libraries/System.Diagnostics.StackTrace/tests/StackTraceTests.cs

```diff
@@ -313,7 +313,9 @@ public void ToString_NullFrame_ThrowsNullReferenceException()
         [MethodImpl(MethodImplOptions.NoOptimization | MethodImplOptions.NoInlining)]
         private static StackTrace Generic<T, U>() => new StackTrace();
 
+        [MethodImpl(MethodImplOptions.NoOptimization | MethodImplOptions.NoInlining)]
         private static StackTrace InvokeIgnoredMethod() => Ignored.Method();
+        [MethodImpl(MethodImplOptions.NoOptimization | MethodImplOptions.NoInlining)]
```

Comment:
Why is the NoOptimization part needed?

Reply:
Don't know. I was only following the example of the others nearby.